lib/tests/test_type_hints.c

```diff
@@ -214,14 +214,21 @@ ParameterizedTestParameters(type_hints, test_double_cast)
                              sizeof(string_value_pairs) / sizeof(string_value_pairs[0]));
 }
 
+static gboolean
+cr_assert_gdouble_eq(gdouble a, gdouble b)
+{
+  return fabs(a-b) < G_MINDOUBLE;
```

Comment:
Shouldn't we use `cr_assert` instead of `return`?

Reply:
omg ?!? well at least it does not fails...
thanks, sure we do.